### DISCUSSION TITLE
Integration / vignette for Cicero Gene Activities

### DISCUSSION THREAD

**Discussion by wudustan** [stuart-lab/signac#583] [Q&A] [answered]
Hi,

I really appreciate the Cicero vignette for Signac but I was wondering if it would be possible to include a section for calculating gene activities with Cicero from a Seurat object as the starting point. 

Some aspects of the Cicero guide for gene activities require elements that aren't generated when converting to cds eg `$num_genes_expressed` metrics in the cds `pData()`.

I was also wondering for gene activities generated by Cicero, would you recommend normalising using `Seurat::NormaliseData()`or the Cicero normalisation?

**Comment by timoast** [comment, marked as answer]
The Cicero vignette on the Signac website (https://satijalab.org/signac/articles/cicero.html) shows how to convert to a CellDataSet object and run the Cicero functions. Since you have a CellDataSet object at that point, you can follow the Cicero documentation: https://cole-trapnell-lab.github.io/cicero-release/docs_m3/
